### ISSUE TITLE
Sidebar link text contrast is very low, making items difficult to read

### ISSUE THREAD

**Issue by thomasboyt** [influxdata/docs.influxdata.com#178]
After the recent update, the sidebar links are now `#79818C` on a `#181C23` background, which is extremely low contrast. This fails the accessibility audit in Chrome's [Accessibility Developer Tools](https://chrome.google.com/webstore/detail/accessibility-developer-t/fpkknkljclfencbdbgkenhalefipecmb?hl=en) extension. Lightening the link colors to `#A5ACB5` (the color of the body text), or another color with at least a `7:1` contrast ratio, would make the sidebar much easier to read. Currently, I have to increase the brightness on my Retina Macbook to comfortably read the sidebar links - I imagine this would be even more painful on lower-resolution screens.

**Comment by rkuchan** [comment]
Fixed by https://github.com/influxdata/docs.influxdata.com/pull/352
